bin/helm-generate-raw-manifests-in-docker

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -eo pipefail
+
+diff_only=false
+
+while true; do
+  case "$1" in
+    -d|--diff-only )
+      diff_only=true
+      shift
+      ;;
+    * )
+      if [ -z "$1" ]; then
+        break
+      else
+        echo "$1 is not a valid option"
+        exit 1
+      fi
+      ;;
+  esac
+done
+
+echo "Building utility image with helm and yq..."
+docker build -f Dockerfile.helm-unit-test -t conjur-k8s-helm-generate-manifests:dev .
+
+echo "Generating raw k8s manifests..."
+docker run --name conjur-k8s-helm-generate-manifests --entrypoint "/conjur-authn-k8s-client/bin/helm-generate-raw-manifests" \
+            conjur-k8s-helm-generate-manifests:dev
+
+set +eo pipefail
```

Comment:
Do we need/want to ignore errors for the `docker cp ...` command?

If we really do want to ignore errors for this command, it might be easier to skip the `-o pipefail` setting at the top of the file (i.e. use `set -e`) since I don't think there is any piping used in the script. Then, for the `docker cp ...` commands here, you'd be able to use `<docker-cp-command> || true`.

Reply:
Refactored based on discussion